### PR TITLE
f-registration@2.4.1 - Add tests for policy links

### PR DESCRIPTION
**Pull request by xander-marjoram**
### Added
- Component tests to check correct URLs for Ts & Cs, Privacy and Cookies policy links.

---

## UI Review Checks

- [ ] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: packages/components/pages/f-registration/test/component/f-registration.component.spec.js [hunk 1]
```diff
@@ -19,4 +19,18 @@ describe('Shared - f-registration component tests', () => {
         // Assert
         expect(registration.canBeClicked(link)).toBe(true);
     });
+
+    forEach([
+        ['termsAndConditionsLink', '/info/terms-and-conditions'],
+        ['privacyPolicyLink', '/info/privacy-policy'],
+        ['cookiesPolicyLink', '/info/cookies-policy']
+    ])
+    .it('should check if the legal documentation is clickable', (linkName, expectedUrl) => {
```
Comment (jamieomaguire) [inline packages/components/pages/f-registration/test/component/f-registration.component.spec.js]: the test looks good, just wonder if the title is a little misleading as we're testing that the resolved URL is a match?

Comment (xander-marjoram) [reply packages/components/pages/f-registration/test/component/f-registration.component.spec.js]: Ah sorry, good catch, copy and paste error

**Review by ashleynolan — DISMISSED** [review]

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/pages/f-registration/test/component/f-registration.component.spec.js [hunk 2]
```diff
@@ -19,4 +19,18 @@ describe('Shared - f-registration component tests', () => {
...
+
+    forEach([
+        ['termsAndConditionsLink', '/info/terms-and-conditions'],
+        ['privacyPolicyLink', '/info/privacy-policy'],
+        ['cookiesPolicyLink', '/info/cookies-policy']
+    ])
+    .it('should take you to the correct URL when clicking the `%s`', (linkName, expectedUrl) => {
+        // Act
+        registration[linkName].click();
+        browser.switchWindow(new RegExp(`^.*${expectedUrl}.*$`));
```
Comment (BanksySan) [inline packages/components/pages/f-registration/test/component/f-registration.component.spec.js]: Is `.*` really the correct token here?

Comment (xander-marjoram) [reply packages/components/pages/f-registration/test/component/f-registration.component.spec.js]: What else might you suggest @BanksySan ?

Comment (xander-marjoram) [reply packages/components/pages/f-registration/test/component/f-registration.component.spec.js]: Had a chat on Slack and have updated

File: packages/components/pages/f-registration/test/component/f-registration.component.spec.js [hunk 3]
```diff
@@ -19,4 +19,21 @@ describe('Shared - f-registration component tests', () => {
...
     });
+
+    forEach([
+        ['termsAndConditionsLink', 'info/terms-and-conditions'],
+        ['privacyPolicyLink', 'info/privacy-policy'],
+        ['cookiesPolicyLink', 'info/cookies-policy']
+    ])
+    .it('should take you to the correct URL when clicking the `%s`', (linkName, expectedUrl) => {
+        // Arrange
+        const expectedUrlRegex = new RegExp(`^http(s?)://localhost:\\d+/${expectedUrl}$`);
```
Comment (BanksySan) [inline packages/components/pages/f-registration/test/component/f-registration.component.spec.js:30]: Could there be a trailing slash?

`^http(s?)://localhost:\\d+/${expectedUrl}\\/?$`

**Review by BanksySan — APPROVED** [review]

File: packages/components/pages/f-registration/test/component/f-registration.component.spec.js (hunk 3 above)
Comment (xander-marjoram) [reply packages/components/pages/f-registration/test/component/f-registration.component.spec.js:30]: Not at the moment. If one is added the test can be amended then

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**